### PR TITLE
Dialog for App Lab users

### PR DESCRIPTION
**Pull request by felipeerias**
Implement a startup dialog for App Lab users, informing them that they should install Wolvic from the Oculus Store.

### CODE REVIEW

File: app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java [hunk 1]
```diff
@@ -334,9 +335,11 @@ protected void onCreate(Bundle savedInstanceState) {
         checkForCrash();
 
         // Show the launch dialogs, if needed.
-        if (!showTermsServiceDialogIfNeeded()) {
-            if (!showPrivacyDialogIfNeeded()) {
-                showWhatsNewDialogIfNeeded();
+        if (!showDeprecatedVersionDialogIfNeeded()) {
```
Comment (svillar) [inline app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java:338]: This workflow forces you to duplicate the usual flow inside that method. I think it'd be better to leave the if chain untouched and call this method unconditionally before.

**Review by svillar — CHANGES_REQUESTED** [review]

File: app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java (hunk 1 above)
Comment (svillar) [reply app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java:338]: You still need to address this I think

Comment (felipeerias) [reply app/src/common/shared/com/igalia/wolvic/VRBrowserActivity.java]: @svillar Updated. I looked into a more complex solution to chain the calls, but this simple one seems to work well.

**Review by svillar — CHANGES_REQUESTED** [review]
There is just one thing missing. You have to run this code only for oculus applab's versions. There is already an AppLab flavour so you should be able to get that information from a BuildConfig option you might want to define

**Review by svillar — APPROVED** [review]
Nice!

**Pull request merged**